docs/provisioning-template/createtemplate.md

```diff
@@ -199,6 +199,27 @@ To learn more about individual setting details check the link below:
 
 **[Managing and controlling settings for a single device](../devices-groups/device-settings.md)**
 
+#### What are Extra Settings for Android?
+
+Enable JSON setting when On, lets you input a vaild JSON code. 
```

Comment:
"Enabling" JSON setting when ON, lets you input "valid" JSON code.

-- should setting be plural too?